azure-functions/acbs-function/apim-mdm.js

```diff
@@ -0,0 +1,36 @@
+require('dotenv').config();
+const axios = require('axios');
+
+const headers = {
+  [String(process.env.APIM_MDM_KEY)]: process.env.APIM_MDM_VALUE,
+};
+
+/**
+ * APIM MDM `GET` generic endpoint invocation call
+ * @param {String} endpoint Endpoint with provided query parameters
+ * @returns Response object
+ */
+const getAPI = async (endpoint) => {
+  const response = await axios({
+    method: 'get',
+    url: `${process.env.APIM_MDM_URL}${endpoint}`,
+    headers,
+  }).catch((err) => ({
+    status: err.response.status ?? err,
+    data: {
+      error: err.response.data.error ?? err,
+    },
+  }));
```

Comment:
The logic in the `catch` has changed slightly from before. It used to be

```
catch((err) => ({
    status: err.response
      ? err.response.status
      : err,
    data: {
      error: err.response
        ? err.response.data.error
        : err,
    },
  }))
```

which looks like it's built to expect that `err.response` can sometimes be `undefined`, but the new logic would throw a `TypeError` if `err.response` is `undefined`.

Has the `err` object changed so that we know that `err.response` is always defined? Or was it never necessary in the first place?

Reply:
Good point, have reverted back.